src/lib/web-component/filter-keyword/filter-keyword-element.js

```diff
@@ -0,0 +1,39 @@
+export class FilterKeyword extends HTMLElement {
+  constructor() {
+    super();
+
+    const shadow = this.attachShadow({ mode: 'open' });
+    const filterContainer = document
+      .querySelector('#filterKeywordTemplate')
+      .content.cloneNode(true);
+
```

Comment:
`filter-keyword.css` is currently empty and not referenced. If you don't plan on using it, remove it from the PR.